src/main/java/com/google/step/servlets/FollowServlet.java

```diff
@@ -106,8 +107,45 @@ public void doDelete(HttpServletRequest request, HttpServletResponse response)
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String followerIdString = request.getParameter("followerId");
+    long followerId;
+    try {
+      followerId = Long.parseLong(followerIdString);
+    } catch (NumberFormatException | NullPointerException e) {
+      response.setStatus(HttpStatusCodes.STATUS_CODE_BAD_REQUEST);
```

Comment:
Let's be consistent and return `SC_BAD_REQUEST` instead of `STATUS_CODE_BAD_REQUEST`? Same everywhere else in this file.

Reply:
And the tests too, please